mgmt/smp/transport/smp_uart/src/smp_uart.c

```diff
@@ -354,42 +360,42 @@ nmgr_uart_rx_char(void *arg, uint8_t data)
         }
     }
     /* failed */
-    nus->nus_rx->omp_len = 0;
+    sus->sus_rx->omp_len = 0;
     m->om_len = 0;
     os_mbuf_free_chain(SLIST_NEXT(m, om_next));
     SLIST_NEXT(m, om_next) = NULL;
     return 0;
 }
 
 void
-nmgr_uart_pkg_init(void)
+smp_uart_pkg_init(void)
 {
-    struct nmgr_uart_state *nus = &nmgr_uart_state;
+    struct smp_uart_state *sus = &smp_uart_state;
     int rc;
     struct uart_conf uc = {
         .uc_speed = MYNEWT_VAL(NMGR_UART_SPEED),
```

Comment:
bin/targets/sensors_test/generated/include/syscfg/syscfg.h:15:49: error: 'MYNEWT_VAL_NMGR_UART_SPEED' undeclared (first use in this function); did you mean 'MYNEWT_VAL_SMP_UART_SPEED'?
 #define MYNEWT_VAL(_name)                       MYNEWT_VAL_ ## _name
                                                 ^
repos/apache-mynewt-core/mgmt/smp/transport/smp_uart/src/smp_uart.c:376:21: note: in expansion of macro 'MYNEWT_VAL'
         .uc_speed = MYNEWT_VAL(NMGR_UART_SPEED),
                     ^~~~~~~~~~

Reply:
removed smp_uart

Reply:
This was for low memory platforms which could not fit shell/did not want shell. I've used it, I would have liked to keep it.